StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentMethodsView.swift

```diff
@@ -214,10 +214,24 @@ class EmbeddedPaymentMethodsView: UIView {
         fatalError("init(coder:) has not been implemented")
     }
 
+    private func logRenderLPMs() {
+        // The user has to scroll through all the payment method options before checking out, so all of the lpms are visible
```

Comment:
This isn't necessarily true @joyceqin-stripe @davidme-stripe. Some of our merchants are rendering Embedded in a separate screen from checkout where tapping a row _immediately_ goes back to the checkout screen or immediately confirms.  

If a PM were rendered off screen in this case, the customer can complete checkout without seeing it.